nft/minting_confirmation_task.go

```diff
@@ -0,0 +1,131 @@
+package nft
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/centrifuge/gocelery"
+
+	"github.com/centrifuge/go-centrifuge/centerrors"
+	"github.com/centrifuge/go-centrifuge/queue"
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+)
+
+const (
+	MintingConfirmationTaskName string = "MintingConfirmationTaskName"
+	TokenIDParam                string = "TokenIDParam"
+	BlockHeight                 string = "BlockHeight"
+)
+
+// PaymentObligationMintedFilterer filters the approved NFTs
+type PaymentObligationMintedFilterer interface {
+
+	// FilterPaymentObligationMinted filters PaymentObligationMinted events
+	FilterPaymentObligationMinted(opts *bind.FilterOpts) (*EthereumPaymentObligationContractPaymentObligationMintedIterator, error)
+}
+
+// MintingConfirmationTask confirms the minting of a payment obligation NFT
+type MintingConfirmationTask struct {
```

Comment:
Let's not export the task. This is not used anywhere outside the package. Also, fields of the struct doesn't need to exported as well